src/analytics/Properties.tsx

```diff
@@ -976,6 +977,9 @@ interface QrScreenProperties {
   [QrScreenEvents.qr_screen_bottom_sheet_link_press]: {
     exchange: string
   }
+  [QrScreenEvents.qr_scanner_open]: {
+    fromScreen: Screens
```

Comment:
we already have a super property for current and previous screen, we shouldn't duplicate this here